integration/integration_test.go

```diff
@@ -437,9 +439,8 @@ func testAuditOn(t *testing.T, suite *integrationTestSuite) {
 			// make sure it's us who joined! :)
 			require.Equal(t, suite.Me.Username, tracker.GetParticipants()[0].User)
 
-			// lets type "echo hi" followed by "enter" and then "exit" + "enter":
-
-			myTerm.Type("\aecho hi\n\r\aexit\n\r\a")
+			// let's type "echo hi" followed by "enter" and then "exit" + "enter":
+			myTerm.Type("echo hi\n\rexit\n\r")
```

Comment:
`\a` suspends the terminal for a second https://github.com/gravitational/teleport/blob/1686a71c8a4a4b6b76b236adbd924436850ef433/integration/terminal_test.go#L79
I don't see the point in waiting.